### PR TITLE
feat(vpcep): the VPC endpoint resource support new fields

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:

The VPC endpoint resource support new fields.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. The new fields contain ip_version, ipv6_address.
2. Only professional VPC endpoint supports the parameters.
```

## PR Checklist

* [x] Tests added/passed.

```
$ HW_REGION_NAME=cn-east-4
$ make testacc TEST="./huaweicloud/services/acceptance/vpcep" TESTARGS="-run TestAccVPCEndpoint_professional"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccVPCEndpoint_professional -timeout 360m -parallel 4
=== RUN   TestAccVPCEndpoint_professional
=== PAUSE TestAccVPCEndpoint_professional
=== CONT  TestAccVPCEndpoint_professional
--- PASS: TestAccVPCEndpoint_professional (262.63s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     262.714s
```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

**Comment by dengyali0125** [comment]
LGTM

File: docs/resources/vpcep_endpoint.md [hunk 1]
```diff
@@ -172,6 +172,21 @@ The following arguments are supported:
...
+* `ip_version` - (Optional, String, ForceNew) Specifies the IP version of the VPC endpoint.
+  The valid values are as follows:
+  + **ipv4**: The VPC endpoint IP address can only be an IPv4 address.
+  + **dualstack**: The VPC endpoint IP address can be an IPv4 address or IPv6 address.
+
+* `ipv6_address` - (Optional, String, ForceNew) Specifies the IPv6 address for accessing the connected VPC
+  endpoint service.
+  
+  -> The IPv6 address must be a subnet of the IPv6 network segment of the subnet associated with the VPC endpoint.
+  <br/>If you not specifies, the IPv6 address generated by the system will be used.
```
Comment (profoundwu) [inline docs/resources/vpcep_endpoint.md]:
```suggestion
* `ipv6_address` - (Optional, String, ForceNew) Specifies the IPv6 address for accessing the connected VPC endpoint service.
   The IPv6 address must be a subnet of the IPv6 network segment of the subnet associated with the VPC endpoint.
   If you not specifies, the IPv6 address generated by the system will be used.
```

**Comment by ruwenqiang123** [comment]
```
$ HW_REGION_NAME=cn-east-4
$ make testacc TEST="./huaweicloud/services/acceptance/vpcep" TESTARGS="-run TestAccVPCEndpoint_professional"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpcep -v -run TestAccVPCEndpoint_professional -timeout 360m -parallel 4
=== RUN   TestAccVPCEndpoint_professional
=== PAUSE TestAccVPCEndpoint_professional
=== CONT  TestAccVPCEndpoint_professional
--- PASS: TestAccVPCEndpoint_professional (252.87s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpcep     252.981s
```

File: huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go [hunk 2]
```diff
@@ -403,3 +404,115 @@ EOF
...
+
+resource "huaweicloud_vpc_subnet" "retest" {
+  vpc_id      = huaweicloud_vpc.retest.id
+  name        = "%[2]s-tf"
+  cidr        = "192.168.0.0/24"
+  gateway_ip  = "192.168.0.1"
+  ipv6_enable = true
+}
+
+resource "huaweicloud_vpcep_endpoint" "test" {
```
Comment (jinyangyang222) [inline huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go:510]: I think it is necessary to test all parameters for this test

Comment (ruwenqiang123) [reply huaweicloud/services/acceptance/vpcep/resource_huaweicloud_vpcep_endpoint_test.go:517]: done

File: docs/resources/vpcep_endpoint.md [hunk 3]
```diff
@@ -172,6 +172,21 @@ The following arguments are supported:
   -> Please refer to [official document](https://support.huaweicloud.com/intl/en-us/usermanual-iam/iam_01_0017.html) for
   the data structure rules of the policy. Just pay attention to the fields `Effect`, `Action` and `Resource`.
 
+* `ip_version` - (Optional, String, ForceNew) Specifies the IP version of the VPC endpoint.
```
Comment (deer-hang) [inline docs/resources/vpcep_endpoint.md:175]: Changing this will create XXX.
Same to field `ipv6_address`.

**Pull request merged**